### ISSUE TITLE
Column header alignment in pathway explorer

### ISSUE THREAD

**Issue by joewandy** [kmcluskey/FlyMet#42]
It seems that in the pathway explorer, the column headers are not aligned with the column bodies. See screenshot below for an example. Should be easy to fix?

![image](https://user-images.githubusercontent.com/2523391/94422678-5eed0c80-017f-11eb-8539-112ff1dda4ad.png)

**Comment by kmcluskey** [comment]
This is odd, I can't replicate it at all - I spent some time making sure these aligned but may have missed something - if you can tell me how this happened I'll have a look.

**Comment by joewandy** [comment]
I'm using the JSON and CSV data here for testing: https://github.com/kmcluskey/FlyMet/tree/master/data. Should I be using them? 
```
 python manage.py initialisedb data/tissues_life_stages_no_whole.csv data\67_peak_cmpd_export_1.json data/67_peak_int_export.json
```

**Comment by kmcluskey** [comment]
Would that make a difference to column alignment? - but no those are old- remember I passed you a new set via slack...I'm sure those are not what you used to get it running?

**Comment by kmcluskey** [comment]
Any js errors?

**Comment by joewandy** [comment]
Yeah I was using them (as above) to get it running yesterday. I'll try with the other data from slack.

**Comment by kmcluskey** [comment]
Okay, let me know, I can't live with that!

**Comment by joewandy** [comment]
Still the same problem with the other data. Tested on Chrome, Firefox and Microsoft Edge. Not sure it makes a difference since I'm running Windows, although I guess not?

**Comment by kmcluskey** [comment]
Any css errors or js in the console?

**Comment by joewandy** [comment]
Saw this ![image](https://user-images.githubusercontent.com/2523391/94671827-4bbb7780-030c-11eb-9bed-82612fbb2034.png)

Not sure what the first error means, but the rest of the script after that seems to run?

**Comment by kmcluskey** [comment]
I have never seen this error before, or noticed it. Odd.
